core/chain_util.go

```diff
@@ -32,7 +32,7 @@ import (
 var (
 	blockHashPre  = []byte("block-hash-")
 	blockNumPre   = []byte("block-num-")
-	expDiffPeriod = big.NewInt(100000)
+	ExpDiffPeriod = big.NewInt(100000)
```

Comment:
Nevermind, I checked the spec and see why package core is guaranteed to always be initialised before cmd :)